decidim-assemblies/lib/decidim/assemblies/participatory_space.rb

```diff
@@ -46,10 +46,9 @@
   end
 
   participatory_space.exports :assemblies do |export|
-    export.collection do
-      Decidim::Assembly
-        .public_spaces
-        .includes(:attachment_collections)
+    export.collection do |participatory_space, _user|
+      query = Decidim::Assembly.public_spaces.includes(:attachment_collections)
+      participatory_space ? query.where(id: participatory_space) : query
```

Comment:
Good catch, however since we got here, we should remove the else statement of the ternary. Reading, i realize that we have the chance to query this, and pass over the organization's scope. If is the case, we should return an empty / nil result. 

```suggestion
      Decidim::Assembly.public_spaces.includes(:attachment_collections).where(id: participatory_space)
```